devito/builtins/arithmetic.py

```diff
@@ -33,11 +33,11 @@ def norm(f, order=2):
     with MPIReduction(f) as mr:
         op = dv.Operator([dv.Eq(s, 0.0)] +
                          eqns +
-                         [dv.Inc(s, Abs(Pow(p, order))), dv.Eq(mr.n[0], s)],
+                         [dv.Inc(s, dv.Abs(Pow(p, order))), dv.Eq(mr.n[0], s)],
```

Comment:
why can we do `dv.Abs` but not `dv.Pow`? this looks odd...

Reply:
because `Pow` is explicitly not imported `not in its own _all__` while `Abs` is not imported but its submodule is so `.Abs` is defined.

Reply:
btw, probably now `p**order` would work too